src/reader.py

```
@@ -549,91 +549,91 @@ def build_solr_query(bsq_query):
                                                                                 re.sub(r'\s+', '', add_port)
                                                                             )                                   + " || "# отбор по дополнительному порту
             # r17 - отбор по сеансу ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
             for seanse                                      in  g.rexp.q.seans_re.findall(param):
-                bsq_seanses                                 +=  "r17:"  +   seanse                              + " || "# отбор по сеансу.
+                bsq_seanses                                 +=  "session_id:"  +   seanse                              + " || "# отбор по сеансу.
             # r18 - отбор по разделению основных данных ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~#case 2020.05.21
             for ext_main_nmb                                in  g.rexp.q.ext_main.findall(param):                       #case 2020.05.21
-                bsq_ext_main                                +=  "r18:"  +   ext_main_nmb                        + " || "# отбор по разделению основных данных.#case 2020.05.21
+                bsq_ext_main                                +=  "area_id:"  +   ext_main_nmb                        + " || "# отбор по разделению основных данных.#case 2020.05.21
             # r19 - отбор по разделению дополнительных данных ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~#case 2020.05.21
             for ext_ext_nmb                                 in  g.rexp.q.ext_add.findall(param):                        #case 2020.05.21
```

Ruff (0.15.17)

[warning] Loop control variable `ext_ext_nmb` not used within loop body

Rename unused `ext_ext_nmb` to `_ext_ext_nmb`

(B007)